### PR TITLE
update calphy interface

### PR DESCRIPTION
**Pull request by srmnitc**
Update calphy interface to support versions >= 1.3

### CODE REVIEW

**Comment by srmnitc** [comment]
@pmrv Still, the job somehow does not collect and populate the output when running on the cluster. Any ideas you have this would also be super useful

**Comment by jan-janssen** [comment]
```
======================================================================
ERROR: test_get_element_list (calphy.test_base.TestCalphy)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/pyiron_atomistics/pyiron_atomistics/tests/calphy/test_base.py", line 139, in test_get_element_list
    pm, pl = self.job._get_masses()
ValueError: not enough values to unpack (expected 2, got 1)
```

File: pyiron_atomistics/calphy/job.py [hunk 1]
```diff
@@ -720,39 +733,45 @@ def collect_general_output(self):
...
+            f_ediff, b_ediff, flambda, blambda = self._collect_ediff(concentration)
             self.output["fe/forward/energy_diff"] = list(f_ediff)
             self.output["fe/backward/energy_diff"] = list(b_ediff)
             self.output["fe/forward/lambda"] = list(flambda)
             self.output["fe/backward/lambda"] = list(blambda)
 
             # get final structure
-            traj = PyscalTrajectory(
-                os.path.join(self.working_directory, "conf.equilibration.dump")
+            aseobj = read(
```
Comment (pmrv) [inline pyiron_atomistics/calphy/job.py]:
```suggestion
            aseobj = self.project.create.structure.read(
```

This should save you the `ase_to_pyiron` call below.

**Comment by pmrv** [comment]
> @pmrv Still, the job somehow does not collect and populate the output when running on the cluster. Any ideas you have this would also be super useful

You mean that once the job on the cluster finishes, that the object in your local notebook doesn't show the output?  That's not really solvable so easily, because we have no communication between the process running on the cluster and the process running the notebook.

**Pull request merged**